packages/e2e-test-utils/src/customFonts.js

```diff
@@ -35,20 +79,23 @@ export const addCustomFont = async (fontUrl) => {
 
   await page.keyboard.type(fontUrl);
   await expect(page).toClick('button', { text: 'Add Font' });
-  // TODO(#10878): Target font by font name.
-  await page.waitForSelector('[aria-label="Remove font"]');
+  await expect(page).toMatch(fontUrl);
 };
 
 /**
  * Remove a custom font from the settings page.
  *
+ * @param {string} fontName fontName to delete
  * @return {Promise<void>}
  */
-export const removeCustomFont = async () => {
-  await visitSettings();
-  // TODO(#10878): Target font by font name.
-  await page.hover('[aria-label="Remove font"]');
-  await page.click('[aria-label="Remove font"]');
+export const removeCustomFont = async (fontName) => {
+  const selector = `[aria-label="Delete ${fontName}"]`;
+  await page.waitForSelector(selector);
+  await page.focus(selector);
+  await page.click(selector);
   await page.waitForSelector('[role="dialog"]');
   await expect(page).toClick('button', { text: 'Delete Font' });
+  // add a small delay seems to take a moment for the font list to catchup
+  // return from dialog + re-focus on list
+  await page.waitForTimeout(1000);
```

Comment:
Can we instead wait for dialog to disappear and for the font to disappear from the list too?

We should avoid `waitForTimeout` where possible

Reply:
Yes --- but there's also that delay for the font list rendering --- which I'm hoping we can get rid of as well

https://github.com/GoogleForCreators/web-stories-wp/pull/10918#discussion_r831231006

Maybe I can wait for the dialog to disappear and check that something on the listbox has focus vs checking the font list.

Reply:
Removed 

https://github.com/GoogleForCreators/web-stories-wp/pull/10918/files#diff-15102bcca0cb4bffec2cea43a88ada5c191041b40f7e784266ac7b5cd2161a1dR102

Now checking count of child nodes